File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Guild.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Guild {

    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("realm")
    @Expose
    public String realm;

}

File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Guild.kt
package fr.jbme.raiderioapp.data.model.character

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Guild(
    @SerializedName("name")
    @Expose
    var name: String? = null,

    @SerializedName("realm")
    @Expose
    var realm: String? = null
)